core/frontend/src/types/services.ts

```diff
@@ -0,0 +1,8 @@
+import { Service } from './common'
+
+export const service_scanner_service: Service = {
+  name: 'Service Scanner',
+  description: 'Actual scanner for running services.',
+  company: 'Blue Robotics',
+  version: '0.1.0',
```

Comment:
How this number will be updated and how we are going to track it ?

Reply:
This comes from the backend Helper service.